go-controller/pkg/ovn/multicast_test.go

```diff
@@ -441,10 +441,15 @@ var _ = ginkgo.Describe("OVN Multicast with IP Address Family", func() {
 				err := fakeOvn.controller.disableMulticast()
 				gomega.Expect(err).NotTo(gomega.HaveOccurred())
 				// check acls are deleted when multicast is disabled
-				dereferenceAllACLs(initialData)
-
-				gomega.Eventually(fakeOvn.nbClient).Should(libovsdb.HaveData(
-					initialData))
+				clusterPortGroup, clusterRtrPortGroup = getDefaultPortGroups()
+				namespacePortGroup := fakeOvn.controller.buildPortGroup(libovsdbutil.HashedPortGroup(namespaceName1), namespaceName1, nil, nil)
+				namespacePortGroup.UUID = namespacePortGroup.Name + "-UUID"
```

Comment:
I'd say this is worth an extra function, since it is used 3 times

Reply:
This pattern was already used many times throughout the tests, not just the three I introduced. I am not really sure a function for a two liner is worth it and ot sure I want to obfuscate the UUID thingy either.